### PR TITLE
refactor #4 Moved the get_markov_blanket method

### LINKED ISSUE #4: Move `BayesianModel.get_markov_blanket` method

### PR DESCRIPTION
**Pull request by agylling**
Fixes #4

### CODE REVIEW

File: pgmpy/base/DAG.py [hunk 1]
```diff
@@ -356,3 +356,36 @@ def get_children(self, node):
...
+        list(blanket_nodes): List of nodes contained in Markov Blanket
+
+        Parameters
+        ----------
+        node: string, int or any hashable python object.
+              The node whose markov blanket would be returned.
+
+        Examples
+        --------
+        >>> from pgmpy.models import BayesianModel
```
Comment (gpihl) [inline pgmpy/base/DAG.py]: The documentation here needs to change, I think all instances of BayesianModel should be replaced by DAG.

**Review by gpihl — CHANGES_REQUESTED** [review]
See comment.

File: pgmpy/base/DAG.py (hunk 1 above)
Comment (agylling) [reply pgmpy/base/DAG.py]: AAh yes you are right, missed that

Comment (agylling) [reply pgmpy/base/DAG.py]: Fixed

**Review by gpihl — APPROVED** [review]

File: pgmpy/base/DAG.py (hunk 1 above)
Comment (gpihl) [reply pgmpy/base/DAG.py]: Looks good.

**Pull request merged**